forwards_paper.tex

```diff
@@ -151,8 +157,12 @@ \section*{Introduction}
 can ``only'' simulate tens of megabases of sequence in tens of thousands of individuals
 for tens of thousands of generations --
 but still substantially short of whole genomes for most biological population sizes.
+\jda{I'd find this more compelling with something added like: "e.g., Xs of
+	gigabases for Ys of individuals, implying 10Ys of generations needed to
+	equilibrate}
```

Comment:
Added; but leaving off the 'to equilibrate' thing since it's useful even without running for N generations - once N is big enough, history starts to be very different.